src/components/App/App.jsx

```diff
@@ -15,6 +15,14 @@ import GrocItemMainComponent from '../MainGrocItemContainer/GrocItemMainComponen
 // Contains ALL the components that comprise our core React app within it.
 function App() {
 
+    // useEffect to execute 'getGrocItems' when App loads.
+    useEffect(() => {
+        getGrocItems();
+    },[]);
+
+    // useState controls the grocItemList array.
+    const [grocItemList, setGrocItemsList] = useState([]);
```

Comment:
Please change `grocItemList` to `grocItemsList`.

This will keep the naming convention the same between the state and the setState values.